### PR TITLE
Pass writeable down to addMarkers

### LINKED ISSUE #721: Support marker-size and marker-symbol simplestyle properties
Before the latest release, there was the ability to edit marker properties — `marker-color`, `marker-size`, `marker-symbol`, or add a custom attribute.  Now it is broken. The Edit tool which appears upon clicking on a marker is read-only now.

### PR DESCRIPTION
**Pull request by wboykinm**
Should resolve #721 

@chriswhong Lemme know what you think here; it feels dangerous to set a parameter to a default like this, but I couldn't find another way to do it.

### CODE REVIEW

**Review by chriswhong — CHANGES_REQUESTED** [review]
This isn't the right fix.  We should be passing `writable` down through `geojsonToLayer()` and then into `addMarkers()`.

I'll push a quick fix to this branch

**Comment by chriswhong** [comment]
We were calling `addMarkers()` without passing in `writable`

**Review by chriswhong — APPROVED** [review]

File: src/ui/map/util.js [hunk 1]
```diff
@@ -61,15 +61,15 @@ const addMarkers = (geojson, map, context, writable) => {
...
 };
-function geojsonToLayer(geojson, map, context) {
-  if (map.isStyleLoaded()) {
-    map.getSource('map-data').setData(addIds(geojson));
+function geojsonToLayer(geojson, context, writable) {
+  if (context.map.isStyleLoaded()) {
+    context.map.getSource('map-data').setData(addIds(geojson));
   
-    addMarkers(geojson, map, context);
+    addMarkers(geojson, context, writable);
```
Comment (wboykinm) [inline src/ui/map/util.js:72]: @chriswhong Man, nicely done finding all the occurrences of this.

**Review by wboykinm — COMMENTED** [review]
GTG

**Comment by wboykinm** [comment]
image

**Pull request merged**